packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -537,28 +744,179 @@ class _ListWheelScrollViewState extends State<ListWheelScrollView> {
             clipToSize: widget.clipToSize,
             renderChildrenOutsideViewport: widget.renderChildrenOutsideViewport,
             offset: offset,
-            children: widget.children,
+            childDelegate: widget.childDelegate,
           );
         },
       ),
     );
   }
 }
 
+/// Element that supports building children lazily for [ListWheelViewport].
+class ListWheelElement extends RenderObjectElement implements ListWheelChildManager {
+  /// Creates an element that lazily builds children for the given widget.
+  ListWheelElement(ListWheelViewport widget) : super(widget);
+
+  @override
+  ListWheelViewport get widget => super.widget;
+
+  @override
+  RenderListWheelViewport get renderObject => super.renderObject;
+
+  // We inflate widgets at two different times:
```

Comment:
Suggestion: explain at the beginning of the comment that _childWidgets is the cache where we store inflated widgets. That will make it clear why that comment is here and not somewhere else.